content/doc/administration/requirements/upgrade-java-guidelines.adoc

```diff
@@ -14,57 +14,57 @@ As with any upgrade, we recommend backing up `JENKINS_HOME` and testing the upgr
 == Upgrading Jenkins
 
 NOTE: In production environments, you should use only LTS versions of Jenkins.
-Java 11 support was first included in a Long Term Support release beginning with Jenkins 2.164.1.
+We first included Java 11 support in a Long Term Support release beginning with Jenkins 2.164.1.
```

Comment:
I thought that using the impersonal form ("Java 11 support was first included") was preferred for most technical writing.
```suggestion
We first included Java 11 support in a Long Term Support release beginning with Jenkins 2.164.1.
```

Reply:
Copywriters avoid using passive voice and using he or she but using we, they, us is not an issue. I can change to "Java 11 support was first included" if you think it's better.
And what about "Long Term Support release beginning with Jenkins 2.164.1." may be "Long Term Support release since Jenkins 2.164.1." is more correct?